src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Amazon SimpleWorkFlow integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon SimpleWorkFlow integration: what data it reports and how to enable it."
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for AWS SimpleWorkFlow, sending its metrics data to New Relic.
+
+This document explains the integration's features, how to activate it, and what data can be reported.
+
+## Features
+
+Use our integration to collect and send telemetry data to New Relic from your [Amazon Simple Workflow Service](https://aws.amazon.com/swf/). You can monitor your services, query incoming data, and build dashboards to observe everything at a glance.
+
+## Activate integration
+
+To enable this integration, see [how to connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
+
+## Find and use data
+
+To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics and events** and filter by `aws.SimpleWorkFlow`.
```

Comment:
```suggestion
## Get started

To enable this integration, see [how to connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).

Once the integration is set up, you can find your metric data by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics and events** and filter by `aws.SimpleWorkFlow`.
```

Reply:
@rahul188 What do you think about this change?

Reply:
@ally-sassman it seems good.